packages/server/graphql/public/types/helpers/getActiveTeamCountByOrgIds.ts

```diff
@@ -0,0 +1,76 @@
+import getRethink from '../../../../database/rethinkDriver'
+import {RValue} from '../../../../database/stricterR'
+import TeamMember from '../../../../database/types/TeamMember'
+import {DataLoaderWorker} from '../../../graphql'
+import isValid from '../../../isValid'
+import {InputMaybe} from '../../resolverTypes'
+import {getTeamsByOrgIds} from '../Company'
+
+const THIRTY_DAYS = 1000 * 60 * 60 * 24 * 30
+
+const getActiveTeamCountByOrgIds = async (
+  orgIds: string | string[],
+  dataLoader: DataLoaderWorker,
+  after?: InputMaybe<Date>
+) => {
+  // teams with at least 2 team members who have logged in within the last 30 days & within an active organization that has had a meeting that has an max(createdAt, endedAt) newer than 30 days ago
+  const metAfter = after ? new Date(after) : new Date(Date.now() - THIRTY_DAYS)
```

Comment:
We have now slightly different algorithms to determine active teams count here and in teams limit. Do you think we should do something about it?

Reply:
How are we currently defining it in teams limits? I agree that it should be consistent across teams limits, Usage Stats, and the org view

Reply:
@nickoferrall for team limits we count sticky teams. Here is the query: 

https://github.com/ParabolInc/parabol/blob/b00aec49e224f499d53ccaca0ef5bf4b5a719b4f/packages/server/billing/helpers/teamLimitsCheck.ts#L85

Sticky team is the team that completed 3 meetings with more than 1 attendee and have had at least 1 meeting in the last 30 days

Reply:
Thanks, @igorlesnenko. Which definition do you think we should change? Where did the teams limit definition come from? 

It looks like the main difference is the Usage Stats definition says the 2 team members logged in within the last 30 days. However, if the Team Limits query is checking if there were 2 team members in the meeting that ended within 30 days, then that's essentially the same thing, right?

If that's correct, then perhaps we just update the Teams Limit definition with the Usage Stats definition, as Usage Stats was the original?

Also, I see the Teams Limit definition says there was `at least 1 meeting in the last 30 days` but I don't see anything in the query that's checking if the meeting did end within the past 30 days?

Reply:
Thanks for explaining that @igorlesnenko. 

> Maybe keep everything as is for now and decide if we need to change after the experiment? 

I agree, I don't think it's something we need to spend time on now as we may change the teams limit after the experiment, e.g. to 3 teams. 

Would you mind creating an issue in the teams limit milestone so that we can revisit this topic once the experiment is complete and if we decide to roll out the current limit to all users?

Reply:
Make sense, created #7831